app.js

```diff
@@ -50,9 +51,26 @@ app.get("/:rideId?", function(req, res){
 });
 
 
+app.get("/rideInfo/:rideId?", function(req, res){
+				mongoose.model('sensorData').find({ "rideId" : req.params.rideId }, function (err, result) {
+			        if (err) {
+								  res.status(404).send;
+									console.log("ERRORED at GET /   , ");
+			            return console.error(err);
+			        } else {
+									console.log("SUCCESS at GET /   , for collection : rideInfo");
+                  console.log("Get results for RIDE ID =  " + req.params.rideId );
+                  res.json({ "result" : result });
+                  //res.render("index.html");
+			        }
+			  });
+
+});
+
 
 
-app.post("/:rideID?", function(req, resp){
+
+app.post("/sensorData/:rideId?", function(req, resp){
```

Comment:
This route might not be needed. The above route (`sensorData/:rideId/:tagId?`) might suffice.